src/Utils.php

```diff
@@ -286,6 +299,7 @@ public static function jsonDecode(string $json, bool $assoc = false, int $depth
      */
     public static function jsonEncode($value, int $options = 0, int $depth = 512): string
     {
+        /** @var string $json */
```

Comment:
This could actually be false. We only know it is a string after the guard has been checked on the lower line. This doc should be moved to just above the return statement.

Reply:
Good point. Fixed in 1cb4840a6dda807ea7a86d1785950fc10f2a7a36